std_msgs/Header header
erau_fusion_msgs/veh_object[32] objects
